File: Binary representation - GFG/binary-representation.cpp
//{ Driver Code Starts
#include<bits/stdc++.h> 
using namespace std; 

// } Driver Code Ends
class Solution
{
public:
    string getBinaryRep(int N)
    {
        // Write Your Code here
        
        int digit = 0;
        string ans= "000000000000000000000000000000";
        int l = 29;
        while(N!=0)
        {
            if(N%2==0){
                ans[l] = '0';
                l--;
            }
            else{
                ans[l] = '1';
                l--;
            }
            N = N/2;
        }
        return ans;
    }
};

//{ Driver Code Starts.
int main()
{
    int t;
    cin >> t;
    while (t--)
    {
        int N;
        cin >> N;
       
        Solution ob;
        string ans = ob.getBinaryRep(N);
        cout << ans << endl;
    }
    return 0;
}
// } Driver Code Ends

File: Binary representation - GFG/binary-representation.java
//{ Driver Code Starts
//Initial Template for Java
import java.io.*;
import java.util.*;

class GFG
{
    public static void main(String args[])throws IOException
    {
        Scanner sc = new Scanner(System.in);
        int t = sc.nextInt();
        while(t-- > 0)
        {
            int N = sc.nextInt();

            Solution ob = new Solution();
            String ans = ob.getBinaryRep(N);
            System.out.println(ans);
        }
    }
}
// } Driver Code Ends


//User function Template for Java
class Solution{
    static String getBinaryRep(int N){
        // code here
        
       String str = "";
       int count = 0;
       while(N!=0){
           str = N%2+str;
           N = N/2;
           count++;
       }
       
       for(int i = 0; i<30-count; i++){
           str = "0"+str;
       }
       
       return str;
    }
}